stories/Components/UIcomponents/Cards/FeaturedCards/FeaturedContentCard.jsx

```diff
@@ -42,14 +45,13 @@ export const FeaturedContentCard = ({
                 {item.size}
                 {item.descriptionText && <P label={item.descriptionText} />}
               </div>
-              <div className={['feature__cta', `${size_variant}`].join(' ')}>
-                <span className="cta__link cta--arrow">{item.button}</span>
-              </div>
+                <span className='cta__link cta--arrow'>{item.button}</span>
```

Comment:
hey @harishkumar-srijan  we can use ctalink component here

Reply:
on CTAlink @Ashish-srij is working, he will change the CTA link component in feature card